build_datacard.py

```diff
@@ -445,6 +452,10 @@ def build_bkg_histograms(args=None):
                     # Handle the case where the number following 'bdt=' is not valid
                     raise ValueError("Invalid number {} following 'bdt='.".format(parts[1]))
      
+                # Apply the signal region
+                col = col.select(col.to_numpy(['rt']).ravel() > 1.18)
```

Comment:
why is this done twice?

Reply:
Indeed, this needs to be cleaned up. I will address it when I update the `build_datacard.py` file

Reply:
Will fix when I rebase on top of your next pull request